G1ANT.Addon.Core/Variables/NewLineVariable.md

```diff
@@ -0,0 +1,26 @@
+# newline
+
+## Syntax
+
+```G1ANT
+⟦text⟧ = ♥newline
+```
+
+## Description
+
+Allows to insert a new line (carriage return and line feed character) into text. It’s equal to this C# snippet:
+
+```G1ANT
+♥altNewLine = ⊂"\r\n"⊃
```

Comment:
Not really. It is `Environment.NewLine`.

Reply:
But this is a variable, so I show how you can make a variable, which does exactly the same